docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,577 @@
+
+    # Make lists to hold the new ground truths, tracks, detections, and clutter
+    new_truths, new_tracks, new_measurements, new_clutter = [], [], [], []
+    for truth in truths_by_time[i]:
+        new_truths.append([truth.state_vector[0], truth.state_vector[2]])
+    for state in tracks_by_time[i]:
+        new_tracks.append([state.state_vector[0], state.state_vector[2]])
+    for measurement in all_measurements[i]:
+        if type(measurement) == TrueDetection:
+            new_measurements.append([measurement.state_vector[0], measurement.state_vector[1]])
+        elif type(measurement == Clutter):
+            new_clutter.append([measurement.state_vector[0], measurement.state_vector[1]])
+
+    # Plot the contents of these lists on the right axis
+    if len(new_truths) > 0: truths.set_offsets(new_truths)
+    if len(new_tracks) > 0: tracks.set_offsets(new_tracks)
+    if len(new_measurements) > 0: measurements.set_offsets(new_measurements)
+    if len(new_clutter) > 0: clutter.set_offsets(new_clutter)
+
+    # Create a legend. The use of Line2D is purely for the visual in the legend
+    data_types = [Line2D([0], [0], color='white', marker='o', markerfacecolor='blue', markersize=15,
+                         label='Ground Truth'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='orange', markersize=15,
+                         label='Clutter'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='green', markersize=15,
+                         label='Detection'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='red', markersize=15,
+                         label='Track')]
+    axR.legend(handles=data_types)
+
+    return sf, truths, tracks, measurements, clutter
+
+# Set up the x, y, and z space for the 3D axis (the left one)
+xx = np.linspace(x_min-5, x_max+5, 100)
+yy = np.linspace(y_min-5, y_max+5, 100)
+x, y = np.meshgrid(xx,yy)
+zarray = np.zeros((100, 100, number_steps))
+
+# Create the matplotlib figure and axes.
+fig = plt.figure(figsize=(10, 6))
+axL = fig.add_subplot(121, projection='3d')
+axR = fig.add_subplot(122)
+axR.set_xlim(x_min-5, x_max+5)
+axR.set_ylim(y_min-5, y_max+5)
+
+# Add an initial surface to the left axis and scattered points on the right axis. Doing
+# this now means that in the animate() function we only have to update these variables
+sf = axL.plot_surface(x, y, zarray[:,:,0], cmap=cm.RdBu, linewidth=0, antialiased=False)
+truths = axR.scatter(x_min-10, y_min-10, c='blue', linewidth=6, zorder=0.5)
+tracks = axR.scatter(x_min-10, y_min-10, c='red', linewidth=4, zorder=1)
+measurements = axR.scatter(x_min-10, y_min-10, c='green', linewidth=4, zorder=0.5)
+clutter = axR.scatter(x_min-10, y_min-10, c='orange', linewidth=4, zorder=0.5)
+
+# Create and display the animation
+from matplotlib import rc
+anim = animation.FuncAnimation(fig, animate, frames = number_steps, interval=500,
```

Comment:
```suggestion
anim = animation.FuncAnimation(fig, animate, frames=number_steps, interval=500,
```
No whitespace needed around keyword assignment operator